tdrs-backend/tdpservice/parsers/case_consistency_validator.py

```diff
@@ -288,3 +391,72 @@ def __validate_s2_records_are_related(self):
                     num_errors += 1
 
         return num_errors
+
+    def __validate_t5_aabd_and_ssi(self):
+        print('validate t5')
+        num_errors = 0
+        is_ssp = self.program_type == 'SSP'
+
+        t5_model_name = 'M5' if is_ssp else 'T5'
+        t5_model = self.__get_model(t5_model_name)
+
+        is_state = self.stt_type == STT.EntityType.STATE
+        is_territory = self.stt_type == STT.EntityType.TERRITORY
+
+        for rpt_month_year, reporting_year_cases in self.record_schema_pairs.sorted_cases.items():
+            t5s = reporting_year_cases.get(t5_model, [])
+
+            for record, schema in t5s:
+                rec_aabd = getattr(record, 'REC_AID_TOTALLY_DISABLED')
+                rec_ssi = getattr(record, 'REC_SSI')
+                family_affiliation = getattr(record, 'FAMILY_AFFILIATION')
+                dob = getattr(record, 'DATE_OF_BIRTH')
+
+                rpt_month_year_dd = f'{rpt_month_year}01'
+                rpt_date = datetime.strptime(rpt_month_year_dd, '%Y%m%d')
+                dob_date = datetime.strptime(dob, '%Y%m%d')
+                is_adult = get_years_apart(rpt_date, dob_date) >= 18
```

Comment:
@jtimpe a couple of suggestions on this:

- age formula: (rpt_month_year - DOB)/365.25 
- adult >= 19 in tanf/tribal/ssp context

Reply:
my apologies @jtimpe re: age calc. thanks for this reference: https://github.com/raft-tech/TANF-app/blob/9195ca173671b7da52ff0b56e1f50ea0f90b7968/tdrs-backend/tdpservice/parsers/util.py#L175C22-L175C28